soccer/src/soccer/strategy/agent/position/offense.hpp

```diff
@@ -36,6 +36,9 @@ class Offense : public Position {
 private:
     bool kicking_{true};
 
+    static constexpr float FINAL_BALL_SPEED{0.0f};
```

Comment:
can we change these constants style to `kFinalBallSpeed` and `kBallDecel` ? also leave a comment explaining they are included here for the sake of kick speed calculation. also, I think we already have a constant related to ball deceleration. check if it's the same or not